restapi.go

```diff
@@ -320,6 +320,23 @@ func (s *Session) UserSettings() (st *Settings, err error) {
 	return
 }
 
+// UserUpdateStatus update the user status
+// status   : The new status (Actual valid status are 'online','idle','dnd','invisible')
+func (s *Session) UserUpdateStatus(status string) (st *Settings, err error) {
```

Comment:
Consider making a type and constants for the valid values, like VerificationLevel, so that we can restrict the values of status.